### ISSUE TITLE
Extraction of REZDAT.WAR fails, "invalid title with quotes" error

### ISSUE THREAD

**Issue by pVVEN** [Wargus/wargus#423]
Similar to my issue with War1gus, the extractor fails on my DOS copy of REZDAT.WAR. First it gives this error:

> Invalid title with quotes
> E:\Games\Wargus /strdat.war

Then this error:

> Extraction failed!
> The following command was used to extract the data (you can run it manually in a console to find out more): @E:\Games\Wargus\wartool.exe -v -r E:\Games\Wargus E:\Games\Wargus

Running that in command prompt (with admin privileges) gave this output followed by the "invalid title with quotes" error popup:

> E:\Games\Wargus>@E:\Games\Wargus\wartool.exe -v -r E:\Games\Wargus E:\Games\Wargus
> Detected expansion DOS CD
> Extract from "E:\Games\Wargus" to "E:\Games\Wargus"
> Please be patient, the data may take a couple of minutes to extract...
> Archive "E:\Games\Wargus/rezdat.war"
> Archive "E:\Games\Wargus/strdat.war"

Here's the MD5 of my REZDAT.WAR file:

65321083C9B21FB6C2F31C6B52DEC4E0

 - OS: Windows 10
 - Version 3.2.0 portable

**Comment by timfel** [comment]
I don't think we support having the data in the same folder

**Comment by timfel** [comment]
Why not point the extractor to your CD?

**Comment by pVVEN** [comment]
I tried moving the file to a different folder as well but encountered the same issue. I used a mounted image of the CD (from which I had originally pulled that REZDAT.WAR file), and it seemed to get through TOD fine. However, when the extractor moved on to pulling the BTDP expansion, it finished with the "extraction failed" error. I tried again with the command prompt, running "wartool.exe -v -r H:\DATA" and that seemed to complete successfully. I have no idea what the problem(s) could have been here. I'm attaching a log file generated when the extraction failed, maybe that will show something. 
[Wargus-extraction.log](https://github.com/Wargus/wargus/files/8238457/Wargus-extraction.log)
